cypress/integration/ete/registration/register.spec.ts

```diff
@@ -216,35 +214,29 @@ describe('Registration flow', () => {
         receivedAfter: timeRequestWasMade,
       },
     ).then((email) => {
-      const body = email.html.body;
-
-      const links = email.html.links ?? [];
+      cy.getEmailDetails(email, /\/reset-password\/([^"]*)/).then(
```

Comment:
do we always do this after mailosaurGetMessage?
it might be worth combining the two 🤔

Reply:
also, de we always call delete at the end?
could be worth combining that too 😄

Reply:
both good ideas, I feel like combining them would need a bit more thought and would be a nice followup PR I could open after!